### PR TITLE
update beefy client with latest changes

### PR DESCRIPTION
**Pull request by Wizdave97**
Updates `beefy-generic-client` package

### CODE REVIEW

File: modules/src/clients/ics11_beefy/client_def.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
-use beefy_client::primitives::{ParachainHeader, ParachainsUpdateProof};
-use beefy_client::traits::ClientState as LightClientState;
 use beefy_client::BeefyLightClient;
+use beefy_client_primitives::ClientState as LightClientState;
```
Comment (blasrodri) [inline modules/src/clients/ics11_beefy/client_def.rs:2]: why not rename the struct?

Comment (Wizdave97) [reply modules/src/clients/ics11_beefy/client_def.rs:2]: The struct comes from `beefy-rs` and in the context of the former it makes sense to call it `ClientState`.

File: relayer-cli/Cargo.toml [hunk 2]
```diff
@@ -40,7 +40,7 @@ tokio = { version = "1.0", features = ["full"] }
 tracing = "0.1.34"
 tracing-subscriber = { version = "0.3.11", features = ["fmt", "env-filter", "json"]}
 eyre = "0.6.8"
-color-eyre = "0.6"
+color-eyre = "0.6.2"
```
Comment (blasrodri) [inline relayer-cli/Cargo.toml:43]: cargo update doesn't do the trick?

File: modules/src/core/ics02_client/handler/update_client.rs [hunk 3]
```diff
@@ -674,15 +674,18 @@ mod tests {
...
+                    block_number,
+                    client_state.latest_beefy_height,
+                )
+                .await
+                .unwrap();
+
+            let mmr_update = client_wrapper
+                .fetch_mmr_update_proof_for(signed_commitment.clone())
+                .await
+                .unwrap();
```
Comment (blasrodri) [inline modules/src/core/ics02_client/handler/update_client.rs:688]: Why not return errors instead and let the client handle this? Or at least some `.expect(...)` with a message saying which invariant was broken

**Review by blasrodri — APPROVED** [review]

File: relayer-cli/Cargo.toml (hunk 2 above)
Comment (Wizdave97) [reply relayer-cli/Cargo.toml:43]: It's just this crate that needed to be updated to fix conflicts with another dependency.

File: modules/src/core/ics02_client/handler/update_client.rs (hunk 3 above)
Comment (seunlanlege) [reply modules/src/core/ics02_client/handler/update_client.rs:688]: I think `unwraps` are fine in tests.

**Review by seunlanlege — APPROVED** [review]

File: modules/src/core/ics02_client/handler/update_client.rs (hunk 3 above)
Comment (Wizdave97) [reply modules/src/core/ics02_client/handler/update_client.rs:688]: This block is inside a test, the test should fail if that unwrap fails.

**Pull request merged**